JustinaSetup.sh

```diff
@@ -47,4 +47,5 @@ sudo apt-get install libzbar-dev
 sudo apt-get purge jackd2 jackd jackd2-firewire libjack-jackd2-dev pulseaudio-module-jack qjackctl
 echo "PLEASE SAY YES WHEN ASKED FOR REAL TIME"
```

Comment:
Avoid messages using caps (upper case only). Caps are to stress important words.